frontend/src/components/feedback/optionButton/OptionButton.style.ts

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from "styled-components";
+
+interface OptionButtonBoxProps {
+  isSelected?: boolean;
+}
+
+export const OptionContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.6rem;
+`;
+
+export const ButtonWrapper = styled.button<OptionButtonBoxProps>`
+  cursor: pointer;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  padding: 0.4rem 0.8rem;
+
+  font: ${({ theme }) => theme.TEXT.semiSmall};
+
+  background-color: ${({ theme }) => theme.COLOR.white};
+  border: 2px dashed ${({ theme }) => theme.COLOR.grey1};
+  border-radius: 18px;
+
+  ${({ isSelected, theme }) =>
+    isSelected &&
+    css`
+      border: 4px solid ${theme.COLOR.primary3};
+    `}
```

Comment:
여기 border 크기가 달라서 때문에 선택이 되면 오른쪽으로 살짝 밀립니다! wrap 때문인 것 같은데 어떻게 해결하면 좋을까요?

Reply:
border 속성 아래처럼 한번에 처리해도 좋을 것 같아요!
그리고 `속성: 내용` 이런 형식을 통일성 있게 가져가는 거 어떤가요?

```suggestion
  border: ${(props) =>
    props.isSelected
      ? `4px solid ${props.theme.COLOR.primary3}`
      : `2px dashed ${props.theme.COLOR.grey1}`};
```

Reply:
> 여기 border 크기가 달라서 때문에 선택이 되면 오른쪽으로 살짝 밀립니다! wrap 때문인 것 같은데 어떻게 해결하면 좋을까요?

이 부분은 outline 속성을 사용해서 해결했습니다!! 휴-편안--

- `border` : 요소의 경계 바로 안쪽에 그려지며, border의 두께는 요소의 전체 크기에 포함되어 두께를 변경하면 요소의 크기가 변한다.
- `outline` : outline은 요소의 경계 바깥쪽에 그려지며, 요소의 크기(박스 모델)에 포함되지 않아 두께를 변경해도 요소의 크기가 변하지 않는다.

Reply:
우왘 좋은거 배워갑니다ㅎㅎㅎㅎ 저도 작업하다가 밀리는 현상 생기면 outline 속성 사용해볼게요!!😁